.travis.yml

```diff
@@ -1,18 +1,22 @@
+os: linux
+dist: bionic
 language: python
 python:
   - 2.7
   - 3.5
   - 3.6
   - 3.7
   - 3.8
+services:
+  - docker
 
 before_install:
 - pip install --upgrade pip
 - pip install flake8  # find Python syntax errors and undefined names
 - flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
 install:
-- docker pull ibmcom/db2express-c
-- docker run --name db2  -p 50000:50000 -e DB2INST1_PASSWORD=password -e LICENSE=accept -d ibmcom/db2express-c db2start
+- docker pull ibmcom/db2
+- docker run --name db2  -p 50000:50000 -e DB2INST1_PASSWORD=password -e LICENSE=accept -d ibmcom/db2 db2start
 - docker ps -as
 - docker exec -it db2 useradd -ms /bin/bash auth_user -p auth_pass
 - docker exec -it db2 su - db2inst1 -c "db2 create db sample"
```

Comment:
```suggestion
```
With the `-e DBNAME=sample` addition to the main docker run command I suggested earlier, this line should no longer be necessary.